pinc/smooth_read_change.inc

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
Please put this new function in `pinc/smoothread.inc` with the rest of the smoothreading functions.

Reply:
Done this. A reason for keeping them separate was that the function is only needed in upload_text.php and the other functions in project.php. But of course this logic would lead to hundreds more small include files.